nixos/modules/services/backup/postgresql-backup.nix

```diff
@@ -28,9 +51,9 @@ in
       };
 
       period = mkOption {
-        default = "15 01 * * *";
+        default = "*-*-* 01:15:00";
```

Comment:
Maybe we should rename this option? The type of both is string old values could break silently renaming the option would make this more obvious.

Reply:
`startAt` could be alternative.

Reply:
Yes, that seems more natural in the new version